drivers/storage/portworx/cloud_storage.go

```diff
@@ -0,0 +1,119 @@
+package portworx
+
+import (
+	"fmt"
+
+	"github.com/libopenstorage/cloudops"
+	"github.com/libopenstorage/cloudops/pkg/parser"
+	corev1alpha1 "github.com/libopenstorage/operator/pkg/apis/core/v1alpha1"
+	"github.com/libopenstorage/operator/pkg/cloudstorage"
+	"github.com/portworx/sched-ops/k8s"
+)
+
+const (
+	storageDecisionMatrixCMName = "portworx-storage-decision-matrix"
+	storageDecisionMatrixCMKey  = "matrix"
+)
+
+type portworxCloudStorage struct {
+	zoneCount     int
+	cloudProvider string
+	namespace     string
+}
+
+func (p *portworxCloudStorage) GetStorageNodeConfig(
+	specs []*corev1alpha1.CloudStorageCapacitySpec,
+	instancesPerZone int,
+) (*cloudstorage.Config, error) {
+	// Get the decision matrix config map
+	cm, err := k8s.Instance().GetConfigMap(storageDecisionMatrixCMName, p.namespace)
```

Comment:
I referred to other places in portworx.go file in this directory, which is why I resorted to use k8s.Instance(). Can move it to controller runtime client?

Is there a reason where we should avoid using a client?

Reply:
Oh I thought we don't use sched ops much in the portworx driver. Probably to create CRD because there some problem with loading CRD scheme. If there are more instances with Get calls we should change them to using runtime client.

The runtime client gets the objects from client go shared cache while the create/updates go to api server. Thus saving some API server calls.

Reply:
Agreed. Will move it to the runtime client.